### PR TITLE
Translation to zh_CN - Group 17

### PR DESCRIPTION
**Pull request by shrektan**
@MichaelChirico As I've mentioned to you on WeChat, I'm not able to run `tools::update_pkg_po(pkgdir = ".")` on my Mac for now. The error message looks like below:

```
  R-zh_CN:........................................................... done.
186 translated messages, 395 untranslated messages.
  R-en@quot:
Error in while (startsWith(lines[current], "\"")) { : 
  missing value where TRUE/FALSE needed
```

Anyway, I think the translation itself should be ok for my part. Would you mind review this PR @KingdaShi ? Thanks.

### CODE REVIEW

**Comment by MichaelChirico** [comment]
Thanks Xianyang!

**Comment by MichaelChirico** [comment]
@KingdaShi please have a look when you can & suggest any possible improvements

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -1700,55 +1700,55 @@ msgid "Argument 'sort' should be logical TRUE/FALSE"
...
-msgstr ""
+msgstr "参数`by`和`by.x/by.y`都提供了值。参数`by`的值会被忽略。"
 
 msgid "A non-empty vector of column names are required for `by.x` and `by.y`."
-msgstr ""
+msgstr "`by.x`和`by.y`必须是非空的列名。"
 
 msgid "Elements listed in `by.x` must be valid column names in x."
-msgstr ""
+msgstr "`by.x`中的元素必须是x的有效列。"
```
Comment (KingdaShi) [inline po/R-zh_CN.po]: "`by.x`中的元素必须是x中的有效列名。"

Comment (KingdaShi) [inline po/R-zh_CN.po]: translating ’column names‘  into ‘列名’ is better? 
from 1729 to 1739

File: po/zh_CN.po [hunk 2]
```diff
@@ -3380,32 +3380,39 @@ msgid ""
...
 msgid ""
 "No non-missing values found in at least one group. Returning '-Inf' for such "
 "groups to be consistent with base"
 msgstr ""
+"至少有一个子集的值全为缺失值。为保持和base包一致，对这种情况将会返回'-Inf'"
 
 #: gsumm.c:937
 msgid "Type 'complex' has no well-defined max"
-msgstr ""
+msgstr "类型'complex'不存在有清楚定义的最大值"
```
Comment (KingdaShi) [inline po/zh_CN.po]: Translating into “类型’complex‘没有明确定义的最大值” is better?

Comment (shrektan) [reply po/zh_CN.po]: Yeah, it's better. Thanks.

File: po/R-zh_CN.po (hunk 1 above)
Comment (shrektan) [reply po/R-zh_CN.po]: changed, thanks.

**Comment by MichaelChirico** [comment]
looks like we're all set!! thanks @shrektan and @kingdashi

**Pull request merged**